tests/StatisticsTest.php

```diff
@@ -377,6 +377,20 @@ public function testVariancePopulationFloats()
         $this->assertSame($expected, $result, '', pow(10, -4));
     }
 
+    /**
+     * Test `variance`
+     *
+     * At least two data points.
+     *
+     * @return void
+     * @expectedException \Oefenweb\Statistics\StatisticsError
```

Comment:
I just saw that this annotation syntax issue is deprecated. Can you use this new syntax?

```
$this->expectException(Complicated::class);
```

Reply:
Yes. It will be deprecated on `PHPUnit 9` version.

And I just notice that following code snippets should be changed, too.

https://github.com/Oefenweb/php-statistics/blob/7014d03ccba1351e620046fc0ea95d3d5db57984/tests/StatisticsTest.php#L299

Reply:
Fixed.